cli/tests/pcluster/validators/utils.py

```diff
@@ -28,3 +30,15 @@ def assert_failure_messages(actual_failures, expected_messages):
     else:
         print(actual_failures)
         assert_that(actual_failures).is_empty()
+
+
+def assert_failure_level(actual_failures, expected_error):
```

Comment:
I don't like so much using a "boolean" variable to check True/False/None values. 

I'd like to use:
* True --> we're expecting an error
* False/None --> we're not expecting an error.

But according to the function name you're checking for the "failure level" so you're trying to identify if there are errors or warnings.

In this case you could use `expected_failure_level` and have:
* FailureLevel.ERROR --> check if error exist
* FailureLevel.WARNING --> check if warning exist (you can reuse the same code, just search for the other level)
* None --> assert there are no error nor warnings, actual failures is empty

Let's talk if needed.

Reply:
I agree, will fix in the next rev